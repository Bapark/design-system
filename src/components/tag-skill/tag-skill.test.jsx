import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import TagSkill from './tag-skill.jsx';

describe('TagSkill', () => {
  it('has default', () => {
    expect(renderer.create(<TagSkill name="Egg" />).toJSON()).toMatchSnapshot();
  });

  describe('name API', () => {
    it('can be set', () => {
      render(<TagSkill name="Bacon" />);
      expect(screen.getByText('Bacon')).toBeDefined();
    });
  });

  describe('level API', () => {
    it('can be set', () => {
      render(<TagSkill name="Avocado" level={1} />);
      expect(screen.getByText('1')).toBeDefined();
    });
  });
});
