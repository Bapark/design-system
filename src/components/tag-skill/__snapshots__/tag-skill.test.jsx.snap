// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagSkill has default 1`] = `
Array [
  <span>
    Skillet
  </span>,
  <span>
     - 
  </span>,
  <span>
    Egg
  </span>,
  <span>
     - 
  </span>,
  <span />,
]
`;
